device-types/Datacom/DM4610-OLT-4GPON+4GX+2XS.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Datacom
+model: DM4610 OLT 4GPON+4GX+2XS
+slug: dm4610-olt-4gpon+4gx+2xs
+part_number: 800.5285.xx
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
+airflow: side-to-rear
+comments: OLT GPON para instalação em rack 19’, 8 interfaces GPON, 8 interfaces GE óticas SFP, 4 interfaces GE RJ45 e 2 interfaces 10GE/GE SFP+. Ventilação
+  removível e dois slots para fonte de alimentação AC ou DC redundantes e hot swap.
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: gigabit-ethernet-1/1/1
```

Comment:
Are these actually written as `gigabit-ethernet-1/1/1`? Please send us a sample output of the configuration if you can. We just want to make sure this matches exactly.

Reply:
![image](https://user-images.githubusercontent.com/57509385/215844426-0748ba4a-2349-4918-8783-7ddaa53fb983.png)


DM4610-8GPON-HW2-18-200# show interface
                            L1 RX      L1 TX      L2 RX      L2 TX
INTERFACE NAME              BANDWIDTH  BANDWIDTH  BANDWIDTH  BANDWIDTH
------------------------------------------------------------------------
gigabit-ethernet-1/1/1      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/2      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/3      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/4      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/5      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/6      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/7      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/8      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/9      0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/10     0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/11     0.0bps     0.0bps     0.0bps     0.0bps
gigabit-ethernet-1/1/12     0.0bps     0.0bps     0.0bps     0.0bps
ten-gigabit-ethernet-1/1/1  1.1kbps    0.0bps     889.2bps   0.0bps
ten-gigabit-ethernet-1/1/2  20.1kbps   190.8kbps  16.0kbps   184.8kbps

Reply:
Perfect, thanks!